egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -205,6 +260,28 @@ def ParseModelConfigVarsFile(var_file):
 
     raise Exception('Error while parsing the file {0}'.format(var_file))
 
+def ParseModelConfigGenericVarsFile(var_file):
```

Comment:
Could you rename this to `ParseGenericModelVarsFile`. The current name is a bit difficult to parse.